packages/types/npm/package.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@roo-code/types",
-	"version": "1.71.0",
+	"version": "1.72.0",
```

Comment:
I just bumped this, mind pulling in main before publishing yours?

Reply:
Fixed.